examples/apt_blueprint/tests/test_action_provider.py

```diff
@@ -0,0 +1,92 @@
+import json
+import uuid
+
+import pytest
+
+from examples.apt_blueprint.app import create_app
+from examples.apt_blueprint.blueprint import description
+from globus_action_provider_tools.data_types import ActionStatusValue
+from globus_action_provider_tools.testing.patches import (
+    flask_blueprint_tokenchecker_patch,
+)
+
+
+@pytest.fixture(scope="module")
+def client():
```

Comment:
Some of this is in the toolkit, but is in a haphazard state. As it exists currently, there are some problems with isolating  test cases since the patches patch the base class entirely, and if there are simultaneous tests running that expect a non-patched TokenChecker there'll be issues. I'm looking to patch(heh) the test helpers so they can be used a little more freely and add documentation to using them.